lib/widgets/layouts/agruments/logged_in_body_args.dart

```diff
@@ -1,12 +1,16 @@
 import 'package:fluffychat/widgets/layouts/agruments/app_adaptive_scaffold_body_args.dart';
 
 class LoggedInBodyArgs extends AbsAppAdaptiveScaffoldBodyArgs {
+  final String? roomIdFromNoti;
```

Comment:
Why do you use `LoggedInBodyArgs`? 
IMO: should use `LoggedInOtherAccountBodyArgs`

Reply:
but wait, why login need to care about `roomId`?